.gitignore

```diff
@@ -13,3 +13,4 @@ results
 node_modules
 
 npm-debug.log
+!adjust_get_current_system_volume_vista_plus.exe
```

Comment:
Without it git just ignores this executable. I am not 100% sure but looks like git ignores `*.exe` by default?